code/datums/spells/lightning.dm

```diff
@@ -0,0 +1,89 @@
+/obj/effect/proc_holder/spell/targeted/lightning
+	name = "Lightning Bolt"
+	desc = "Throws a lightning bolt at the nearby enemy. Classic."
+	charge_type = "recharge"
+	charge_max	= 300
+	clothes_req = 1
+	invocation = "UN'LTD P'WAH!"
```

Comment:
Not magicky enough, sounds like a whispered cult spell.

If you disagree, post vocaroo of you screaming this at the top of your lungs while running.